src/components/NewsletterForm/_newsletter-form.scss

```diff
@@ -43,6 +54,38 @@
   order: 4;
 }
 
+/* Errors */
+
+.newsletter-form__item-error {
+  background-color: #ffebeb;
+  background-image: none;
+  border-radius: calc(0.25 * var(--space-unit));
+  color: #d2373f;
```

Comment:
colour variable needed

Reply:
Ticket added for this https://github.com/wellcometrust/corporate/issues/6018